math/PrimeCheck.cpp

```diff
@@ -0,0 +1,30 @@
+#include <iostream>
+using namespace std;
+
+int main()
+{
+    int x,y,z,a=0;
+    // This program will check if a number is a prime number
+    cout << "Number= ";
+    cin >> x;
+
+    for(y=2;y<=(x/2);y++)
+    {
+        z=x%y;
+        if(z==0)
+        {
+            a=a+1;

```

Comment:
This line is unnecessary.If this case occurs then you can print ```"Not a prime number"```
Tip: Use a while-loop